tests/unitTests/finiteVolume/compressibleFlowlesDiffusionTests.cpp

```diff
@@ -0,0 +1,220 @@
+#include <petsc.h>
+#include <cmath>
+#include <map>
+#include <memory>
+#include <vector>
+#include "MpiTestFixture.hpp"
+#include "PetscTestErrorChecker.hpp"
+#include "convergenceTester.hpp"
+#include "domain/boxMesh.hpp"
+#include "domain/modifiers/distributeWithGhostCells.hpp"
+#include "domain/modifiers/ghostBoundaryCells.hpp"
+#include "environment/runEnvironment.hpp"
+#include "eos/mockEOS.hpp"
+#include "eos/transport/constant.hpp"
+#include "finiteVolume/boundaryConditions/essentialGhost.hpp"
+#include "finiteVolume/boundaryConditions/ghost.hpp"
+#include "finiteVolume/compressibleFlowFields.hpp"
+#include "finiteVolume/lesCompressibleFlowSolver.hpp"
+#include "finiteVolume/processes/speciesTransport.hpp"
+#include "gtest/gtest.h"
+#include "mathFunctions/functionFactory.hpp"
+#include "monitors/solutionErrorMonitor.hpp"
+#include "parameters/mapParameters.hpp"
+#include "solver/timeStepper.hpp"
+#include "utilities/petscOptions.hpp"
+#include "utilities/petscUtilities.hpp"
+
+typedef struct {
+    PetscReal L;
+    PetscReal diff;
+    PetscReal rho;
+} InputParameters;
+
+struct CompressibleFlowlesDiffusionTestParameters {
```

Comment:
Can you camel case this CompressibleFlowlesDiffusionTestParameters to CompressibleFlowLesDiffusionTestParameters